curriculum/challenges/english/25-front-end-development/workshop-tailwind-pricing-component/686386e76197458b7dfd52f0.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 686386e76197458b7dfd52f0
+title: Step 7
```

Comment:
While I am all for repetition, I fear that campers will start to zone out if they have to build out two more HTML components before they get to the actual tailwind part. 

My proposal is to just give them the remaining HTML with the exception of the `Most Popular div`. So step 7 could tell them that the rest of the HTML is provided but they just need to add one part. or something like that. 

then we can get to the tailwind stuff quicker. because I am not to considered with their HTML chops since they will already have background in that.